episodes/choosing-acquisition-settings.md

```diff
@@ -0,0 +1,546 @@
+
+## Field of view
+
+The field of view is the size of the area you can view with your light 
+microscope. It is often measured as a diameter in millimetres. A larger field of 
+view is useful if you want to, for example, observe a large number of cells at 
+the same time.
+
+The field of view is affected by many factors but, importantly, is directly 
+related to the magnification of the objective lens. Higher levels of 
+magnification result in a smaller field of view.
+
+To image areas larger than the field of view, you will need to acquire multiple 
+images at different positions by moving the microscope's stage. These images can 
+then be stitched together (often with the microscope manufacturer's acquisition 
+software), to give one large final image. This method can provide high 
+resolution images of large areas, but also slows down overall acquisition times.
+
+## Data size
+
+It's important to consider the overall size of your imaging data (e.g. in 
+megabytes (MB), gigabytes (GB) or terabytes (TB)). Light microscopy datasets can 
+be extremely large in size, so you will have to make sure you have appropriate 
+file storage space that is regularly backed up.
+
+The overall size of your data will depend on many factors. For example, 
+increasing your resolution will increase the size of your final images, 
+especially if you are covering a large area by stitching in x/y.
+
+## Light exposure (i.e phototoxicity / photobleaching)
+
+Illuminating our samples with light is an essential part of collecting light 
+microscopy images. This being said, we have to be very careful of the 
+detrimental effects light can have on our samples, especially at high intensity 
+over long time periods. 
+
+Photobleaching is a key issue for fluorescence microscopy. If a fluorophore is 
+exposed to intense light, its structure can degrade over time. This means that 
+it will eventually stop fluorescing entirely.
+
+Another issue is phototoxicity. Light exposure can damage cells resulting in 
+unexpected changes in cell behaviour, morphology and eventually cell death.
+
+Reducing the effects of photobleaching and phototoxicity requires minimising the 
+light exposure of our samples as much as possible. For example, reducing 
+light/laser power, reducing exposure time/dwell time, imaging at lower temporal 
+resolution...
+
+## Signal to noise ratio
+
+Signal to noise ratio is a very useful measure of image quality - in general, it 
+will be easier to identify and measure features of interest in images with a 
+higher signal to noise ratio.
+
+The 'signal' is what we really want to measure - for example, for a fluorescence 
+microscopy image, this would be the light emitted by fluorophores in the sample. 
+In an ideal world the detector would perfectly measure the light intensity, but 
+this isn't really possible. The values the detector records will always be 
+affected by random fluctuations called 'noise'. This noise usually appears as a 
```

Comment:
Not all noise is random so this statement isn't completely accurate. This paragraph could be expanded to talk about different types of noise. I'm not sure how helpful that would be though.

Reply:
I'll add a link to the breakout, to make it clearer where this information is